src/state/slices/portfolioSlice/portfolioSlice.test.ts

```diff
@@ -30,42 +33,91 @@ const account = {
       }
     ]
   },
-  network: NetworkTypes.MAINNET,
-  pubkey: '0x934be745172066EDF795ffc5EA9F28f19b440c63',
-  symbol: 'ETH'
+  pubkey: '0x934be745172066EDF795ffc5EA9F28f19b440c63'
+}
+
+const btcAccount = {
+  balance: '27803816548287370',
+  caip2: btcCaip2,
+  caip19: btcCaip19,
+  chain: ChainTypes.Bitcoin,
+  chainSpecific: {
+    addresses: [
+      {
+        balance: '1000',
+        pubkey: 'bc1qr9y9lxpynxm8nkswez555xnv2plwwluxrpa55l'
+      },
+      {
+        balance: '0',
+        pubkey: 'bc1q3fmp9tdacg5edlgmh8ttxz7cvj598dcn7w9xxd'
+      },
+      {
+        balance: '10',
+        pubkey: 'bc1qvzuvxskhr5eyaf65w37jxwwvskwyw3rlnqtyzc'
+      },
+      {
+        balance: '0',
+        pubkey: 'bc1q4cqvc3ul562uuz358y77hmqhlfex8jhvfzzek8'
+      }
+    ],
+    nextChangeAddressIndex: 3,
+    nextReceiveAddressIndex: 3
+  },
+  pubkey:
+    'zpub6qk8s2NQsYG6X2Mm6iU2ii3yTAqDb2XqnMu9vo2WjvqwjSvjjiYQQveYXbPxrnRT5Yb5p0x934be745172066EDF795ffc5EA9F28f19b440c637BaBw1wowPwbS8fj7uCfj3UhqhD2LLbvY6Ni1w'
 }
 
+const ethAccountSpecifier = `${ethCaip2}:${ethAccount.pubkey.toLowerCase()}`
+const btcAccountSpecifier = `${btcCaip2}:${btcAccount.pubkey.toLowerCase()}`
```

Comment:
just for the sake of being explicit we shouldn't lowercase btc stuff here, eth is ok to lowercase